### ISSUE TITLE
Additional \ showing in the docs

### ISSUE THREAD

**Issue by redjonzaci** [superfly/docs#978]
If this is something I can fix, let me know!

<img width="471" alt="image" src="https://github.com/superfly/docs/assets/73707194/c34975e9-27e1-49cb-9c80-ee36fbc44973">

**Comment by andie787** [comment]
A few of these seem to left over! If you want to open a docs PR, feel free. One of us can test it out.

**Comment by redjonzaci** [comment]
I just figured out how to do it.

**Comment by redjonzaci** [comment]
I have 2 more suggestions:
- renaming the page where I found the backslashes (https://fly.io/docs/elixir/getting-started/) to "Getting Started with Elixir" for better SEO.
- opening "Edit this page with GitHub" links in a new tab because the contributor might need to compare the page with the GitHub representation (I needed to do that).

If you decide to consider the suggestions, I can work on them, maybe even in this PR, or new PRs.
